### DISCUSSION TITLE
Intermittent loss of audio over AP2 from ATV 4K

### DISCUSSION THREAD

**Discussion by omeroman0** [mikebrady/shairport-sync#1542] [Q&A]
Hi, and thank you for the amazing tool.

I have been running an instance of the dev branch of shairport-sync for a few months now on my Pi 3B+. I use it to "route" the audio from my wired ATV 4K connected to a projector to my old sound bar. It's worked flawlessly until a few weeks ago when I noticed that audio will intermittently be "lost" where for a few seconds it's as if it is muted, interspersed with video with audio. There was also an instance where the audio was out of sync, which a restart corrected.

I have updated the pi, removed and reinstalled shairport following the documentation, and even updated tvOS, but the problem persists. Nothing in my setup has changed, so I am at a loss as to why it is happening.

Any help is appreciated. Thank you.

**Comment by mikebrady** [comment]
Thanks for the post, but this should really be an issue. Perhaps you'd be good enough to open an issue for this.
